src/tox_wheel/plugin.py

```diff
@@ -61,15 +62,21 @@ def patch(obj, attr, value):
 
 @hookimpl
 def tox_package(session, venv):
-    if hasattr(session, "package"):
```

Comment:
Shouldn't this part still be here?

Reply:
It is as it's moved to be an elif below the wheel consideration as it looked like `session.package` is set if it _ever_ builds the sdist.  The key is to imagine a binary package that is generating binary wheels for some interpreters, but otherwise wants an sdist available for platforms they may not have support to build wheels for themselves (i.e. not being able to build for windows or OSX for example).    If you get the order wrong in any way where any sdist is built before any of the wheels, then the remaining wheel builds/tests will no longer considering building a wheel even if it explicitly says in the envconfig that it should be building and testing a wheel